### DISCUSSION TITLE
how to get solver status?

### DISCUSSION THREAD

**Discussion by temph2020** [skfolio/skfolio#85] [Q&A] [answered]
After solving, I want to get solver staus, such as optimal, infeasible,etc. How?

**Comment by HugoDelatte** [comment, marked as answer]
You can do this by setting the parameter `save_problem` to True and accessing it via the attribute `problem_` (then accessing the status using `model.problem_.status`)
You can also set the solver verbosity to True using `solver_params=dict(verbose=True)`.
All of this is explained in the docs: https://skfolio.org/generated/skfolio.optimization.MeanRisk.html

**Reply by temph2020** [reply]
it works, thank you.
